FogCreek/Glitch-Community
# Worried girl: modularize /secret styles

## Links
* Remix link: https://worried-girl.glitch.me

## GIF/Screenshots:
![Screen Shot 2019-06-18 at 1 46 17 PM](https://user-images.githubusercontent.com/938722/59707001-b7d5a300-91cf-11e9-9ab1-0e924b95e105.png)

## Changes:
* modularize styles for /secret page

## How To Test:
* visit https://worried-girl.glitch.me/secret
* verify the page renders correctly and the dev toggles work
